### ISSUE TITLE
Rubustness against Rotation

### ISSUE THREAD

**Issue by adamas-v** [MagicLeapResearch/SuperPointPretrainedNetwork#22]
seem like that the pretrained model privided is not rubust agsinst Rotation,Right？

**Comment by Vincentqyw** [comment]
Yes, SuperPoint+NN can't handle in-plane ratation well, maybe because there are no extreme in-plane rotation in the training examples. In superpoint paper, the author mentioned that they avoid sampling extreme in-plane rotations as they are rarely seen in HPatches.
